arbitrum-docs/stylus/stylus-gentle-introduction.md

```diff
@@ -0,0 +1,120 @@
+- Rich language support already exists for Rust: developers can use the Stylus SDK and CLI tool to **start building today.**
+- Arbitrum's full **EVM equivalence would be maintained** while adding a second, coequal WASM virtual machine.
+- Due to the superior efficiency of WASM programs, Stylus contracts are over **an order of magnitude faster with significantly lower gas fees**.
+- **Memory is 100-500x cheaper** in Stylus, enabling new use cases now that consuming RAM is viable on the blockchain.
+
+### What's Stylus?
+
+Stylus is an upgrade to Arbitrum Nitro, the tech stack powering Arbitrum One, Arbitrum Nova, and Arbitrum Orbit chains. This upgrade adds a second, coequal virtual machine to the EVM, where EVM contracts continue to behave exactly as they would in Ethereum. We call this paradigm \*EVM**+\*** since **everything is entirely additive.**
+
+![Stylus gives you EVM+](./assets/stylus-gives-you-evm-plus.png)
+
+This second virtual machine executes WebAssembly (WASM) rather than EVM bytecode. WASM is a modern binary format popularized by its use in major web standards, browsers, and companies to speed up computation. Built to be fast, portable, and human-readable, WASM also has sandboxed execution environments for security and simplicity. Working with WASM is nothing new for Arbitrum chains. Ever since the [Nitro upgrade](https://medium.com/offchainlabs/arbitrum-nitro-one-small-step-for-l2-one-giant-leap-for-ethereum-bc9108047450), WASM has been a fundamental component of Arbitrum's fully functioning fraud proofs.
+
+With a WASM VM, any programming language that can compile down to WASM are within the bounds of Stylus. While many popular programming languages can be compiled into WASM, some compilers are more suitable for smart contract development than others, like with Rust, C, and C++. Other languages, such as Go, Sway, Move, and Cairo, can be supported as well. Languages that include their own runtimes, like Python and Javascript, are harder for Stylus to support, although not impossible. Third-party contribution in the form of libraries for new and existing languages is welcomed!
+
+Compared to using Solidity, WASM programs are much more efficient. There are many reasons for this, including the decades of compiler development for Rust and C. WASM also has a faster runtime than the EVM, resulting in faster execution. Generally, a **10x improvement** has been seen for contracts using WASM languages compared to contracts using Solidity.
+
+### How is this possible?
+
+Stylus is only possible because of Arbitrum Nitro's unique fraud-proving technology. When there is a dispute on an Arbitrum network, Nitro replays the execution of the chain **in WASM.** Honest Arbitrum validators will then bisect what is being disputed until a single invalid step is identified and checked on-chain via a [“one-step proof.”](https://docs.arbitrum.io/proving/challenge-manager#general-bisection-protocol)
```

Comment:
```suggestion
Stylus is only possible because of Arbitrum Nitro's unique fraud-proving technology. When there's a dispute on an Arbitrum network, Nitro replays the execution of the chain **in WASM.** Honest Arbitrum validators will then bisect what is being disputed until a single invalid step is identified and checked on-chain via a [“one-step proof.”](https://docs.arbitrum.io/proving/challenge-manager#general-bisection-protocol)
```

voice nit -> friendly contractions as a way to "write like we're speaking to a friend"